src/utils/actions.ts

```diff
@@ -1,7 +1,7 @@
 import pipe from 'lodash/fp/pipe';
 import { UniqueActionType } from '~redux';
 
-export type ActionTransformFnType = (
+export type ActionTransformFnType = () => (
```

Comment:
I don't think this is correct. Transform gets called with the action as an argument.

Reply:
I remember that at some point typescript was giving me an error because of `useCallback(pipe...)` but we are removing `useCallback` nevertheless